File: 1450. Number of Students Doing Homework at a Given Time/solution.js
/**
 * @param {number[]} startTime
 * @param {number[]} endTime
 * @param {number} queryTime
 * @return {number}
 */
 var busyStudent = function(startTime, endTime, queryTime) {
  let totalLength = startTime.length;
  let count = 0;
  for (let i = 0; i < totalLength; i++){
    let start = startTime[i];
    let end = endTime[i];
    if (start <= queryTime && queryTime <= end){
      count += 1;
    }
  }
  
  return count;
};

File: 1450. Number of Students Doing Homework at a Given Time/solution.cpp
class Solution {
public:
  int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
    int totalLength = startTime.size();
    int count = 0;
    for(int i = 0; i < totalLength; i++){
      int start = startTime[i];
      int end = endTime[i];
      if (start <= queryTime && queryTime <= end){
        count += 1;
      }
    }

    return count;
  }
};